src/apps/persona/models/entity.py

```diff
@@ -0,0 +1,168 @@
+from enum import IntEnum
+from typing import List, Optional
+
+from sqlalchemy import Column, text
+from sqlalchemy.dialects import mysql
+from sqlalchemy.orm import relationship
+from sqlalchemy.schema import ForeignKey
+from sqlalchemy.types import Enum
+
+from ....core.database import Base
+from ...oauth.entity import User
+from .domain import QuestionCategory, QuestionType
+
+
+class Url(IntEnum):
+
+    max_length = 500
+
+
+class Contents(IntEnum):
+    max_length = 100
+
+
+class Title(IntEnum):
+    max_length = 50
+
+
+class ChoiceItem(Base):
+    """ 선택 항목 """
+
+    __tablename__ = "persona_choices"
+    __table_args__ = {"mysql_collate": "utf8mb4_unicode_ci"}
+
+    uid: Optional[int] = Column(
+        "uid",
+        mysql.BIGINT(20, unsigned=True),
+        primary_key=True,
+        comment="고유 식별자",
+    )
+
+    contents: str = Column(
+        "contents",
+        mysql.VARCHAR(Contents.max_length),
+        nullable=False,
+        comment="선택지 내용",
+    )
+    category: QuestionCategory = Column(
+        "category",
+        Enum(QuestionCategory),
+        nullable=False,
+        comment="집 선택 시 더 가치를 두는 카테고리",
+    )
+
+    question_id: int = Column(
+        "question_id",
+        ForeignKey(
+            "persona_questions.uid", ondelete="CASCADE", onupdate="CASCADE"
+        ),
+        nullable=False,
+        comment="질문지 고유 식별자",
+    )
+    question: Optional["Question"] = relationship(
+        "Question",
+        uselist=False,
+        primaryjoin="ChoiceItem.question_id==Question.uid",
+        backref="persona_choices",
+    )
+
+    image_url: str = Column(
+        "image_url",
+        mysql.VARCHAR(Url.max_length),
+        nullable=False,
+        comment="일러스트 url",
+    )
+
+    def __init__(
+        self,
+        contents: str,
+        category: QuestionCategory,
+        question_id: int,
+        image_url: str,
+        question: Optional["Question"] = None,
+        uid: Optional[int] = None,
+    ) -> None:
+        self.contents = contents
+        self.category = category
+        self.question_id = question_id
+        self.image_url = image_url
+        self.question = question
+        self.uid = uid
+
+
+class QuestionAnswer(Base):
+    """ 문제에 대한 선택 """
+
+    __tablename__ = "persona_answers"
+    __table_args__ = {"mysql_collate": "utf8mb4_unicode_ci"}
+
+    user_id: int = Column(
+        "user_id",
+        ForeignKey("users.uid", ondelete="CASCADE", onupdate="CASCADE"),
+        primary_key=True,
+    )
+    respondent: Optional[User] = relationship(
+        "User",
+        uselist=False,
+        primaryjoin="QuestionAnswer.user_id==User.uid",
+        backref="persona_answer",
+    )
+
+    choice_id: int = Column(
```

Comment:
P4; emptyline을 빼 주셔야 통일성 있을 것 같네요

```suggestion
    choice_id: int = Column(
```

Reply:
연관이 없는 컬럼은 emptyline으로 분리하고 relationship은 위의 variable과 깊은 연관이 있으므로 붙였습니다.